site/content/agent-app-widgets/_index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Agent App Widgets"
+title: "Extending the Agent App integration interface"
```

Comment:
I'm not sure if `integration` is needed here - wouldn't `Extending the Agent App interface` be sufficient?

Reply:
The interface has special meaning in computer science; I doubt if developers will understand what it means.

Reply:
Changing the title from Widgets => Interface was suggested by @fjaskolski, so I guess he needs to comment on this one :)